### ISSUE TITLE
increasing app size about 6MB

### ISSUE THREAD

**Issue by sbrnag** [fechanique/cordova-plugin-fcm#75]
When i use this plugin in my ionic app size got increased about 6MB which i think not usual.

**Comment by fechanique** [comment]
yeah, it's because of the virus I put inside, now you are all infected

**Comment by fechanique** [comment]
just jocking, it's the fcm library
